src/turnkeyml/sequence/sequence.py

```diff
@@ -110,6 +161,20 @@ def launch(self, state: State) -> State:
                 # Save the state so that it can be assessed for a cache hit
                 state.save()
 
+            except exp.SkipBuild as e:
+                # Do not modify the stats or build status for skipped builds
+
```

Comment:
```suggestion
```

Reply:
There was an important nuance here. I added a much longer comment to make sure it is clear.